test/functional/pages/manage-barrister/manage-barrister.js

```diff
@@ -7,7 +7,7 @@ function  manageBarristerApplicant() {        //Matrimonial Case
   I.checkOption('input[id="barristerParty-applicant"]');
   I.waitForContinueButtonEnabled();
   I.click('Continue');
-  I.waitForText('Applicant\'s Barristers');
+  I.waitForText('Applicant');
```

Comment:
is it not better to look for Barrister? or would there be an option to use wildcard like Applicant*Barrister?

Reply:
This is just because "barrister" appears on the previous page, whilst "applicant" does not. So seems a little odd but should be just as effective from a testing perspective.